internal/super/flix.go

```diff
@@ -234,9 +318,67 @@ func getTemplate(state *flowkit.State, flixService flixkit.FlixService, flixQuer
 		if err != nil {
 			return nil, fmt.Errorf("could not parse flix from file %s: %w", flixQuery, err)
 		}
+	case flixUrl:
+		resp, err := http.Get(flixQuery)
```

Comment:
This logic already occurs in flixkit. Why are we duplicating it here? We should be using what flixkit already has

Reply:
now that you mention it, I see that flixkit has `FetchFlixWithContext` which isn't part of the Flix server implementation. Sure I can use `FetchFlixWithContext` to reduce code duplication.